compiler/Makefile

```diff
@@ -80,7 +74,9 @@ $(DISTDIR).tgz: MANIFEST
 	$(RM) -r $(DISTDIR)
 	../scripts/distribution $(DISTDIR)/compiler $<
 	cp -riv ../eclib $(DISTDIR)/
+	mkdir $(DISTDIR)/scripts && cp ../scripts/install-sh $(DISTDIR)/scripts/
```

Comment:
I think the correct fix is to stop relying on `install-sh` to create a directory & copy a few files.

Reply:
Use `cp` then? Or `intall`?

Reply:
I rebased on latest main (and thus on top of https://github.com/jasmin-lang/jasmin/pull/1222).